src/components/EachReview.jsx

```diff
@@ -0,0 +1,47 @@
+import { useParams } from "react-router";
+import { fetchReviewById } from "../../utils/utils";
+import { useEffect, useState } from "react";
+import Comments from "./Comments";
+
+const EachReview = () => {
+  const { review_id } = useParams();
+  const [singleReview, setSingleReview] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchReviewById(review_id).then(({ review }) => {
+      setSingleReview(review);
+      setIsLoading(false);
+    });
+  }, [review_id]);
+
+  if (isLoading) return <p>Loading....</p>;
+
+  let date = new Date(singleReview.created_at);
+  let dd = String(date.getDate()).padStart(2, "0");
+  let mm = String(date.getMonth() + 1).padStart(2, "0"); //January is 0!
+  let yyyy = date.getFullYear();
+
+  let formattedDate = mm + "-" + dd + "-" + yyyy;
```

Comment:
This could be extracted out into a util function in the future if you wanted to! It's fine for now though.